### PR TITLE
Docs: Minor documentation improvements

### PR DESCRIPTION
**Pull request by jonniebigodes**
With this pull request, the Readme and Development instructions were updated to clarify the addon usage.

@tmeasday, when you have a moment, can you look at it and follow up with feedback on this? Also, one quick question for you. Do we want to keep the instructions documented in the `Development` file, or should we instead move them to an internal document?

Let me know, and we'll go from here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,102 @@
...
+  framework: '@storybook/your-framework',
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
+  addons: [
+    // Other Storybook addons
+    '@chromaui/addon-visual-tests',
+  ],
+};
 
+export default config;
 ```
```
Comment (tmeasday) [inline README.md:37]: Should we just recommend people run `npx sb@latest add @chromaui/addon-visual-tests` for these two steps?

File: README.md [hunk 2]
```diff
@@ -1,37 +1,102 @@
...
+
+### Configuration
+
+By default, the addon offers zero-config support to run visual tests with Storybook and Chromatic. However, you can extend your Storybook configuration file (i.e., `.storybook/main.js|ts`) and provide additional options to control how tests are run. Listed below are the available options and examples of how to use them.
+
+
+| Option            | Description                                                                                                                      |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `buildScriptName` | Optional. Defines the custom Storybook build script <br/> `options: { buildScriptName: 'deploy-storybook' }`                     |
+
```
Comment (tmeasday) [inline README.md:49]: I think we should document `projectId`/`projectToken`

**Review by tmeasday — CHANGES_REQUESTED** [review]
Great stuff @jonniebigodes. Couple suggestions

File: README.md (hunk 1 above)
Comment (jonniebigodes) [reply README.md:37]: I thought of it but didn't have the time to test it and see if the command works. Hence I went with this approach. I'll test it out shortly and if nothing stands out, I'll update this part with it

File: README.md (hunk 2 above)
Comment (jonniebigodes) [reply README.md:49]: Good call on this. And yes, we should.

File: README.md (hunk 1 above)
Comment (jonniebigodes) [reply README.md]: Updated with the latest commit.

File: README.md (hunk 2 above)
Comment (jonniebigodes) [reply README.md]: Updated with the latest commit

**Review by tmeasday — COMMENTED** [review]
LGTM

**Review by tmeasday — APPROVED** [review]
LGTM

**Pull request merged**